cmd/nerdctl/compose_version.go

```diff
@@ -0,0 +1,64 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/containerd/nerdctl/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func newComposeVersionCommand() *cobra.Command {
+	var composeVersionCommand = &cobra.Command{
+		Use:           "version",
+		Short:         "Show the Compose version information",
+		Args:          cobra.NoArgs,
+		RunE:          composeVersionAction,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+	composeVersionCommand.Flags().StringP("format", "f", "pretty", "Format the output. Values: [pretty | json]")
```

Comment:
Could you update the shell completion too?

Reply:
This is my first (functional) PR on `nerdctl`, could you elaborate? What would be the candidates?

Reply:
see https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/version.go#L41-L43